ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/scribble/v1/scribble/README.md

```diff
@@ -0,0 +1,99 @@
+<!--
+Copyright 2025 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Adaptive Form Scribble (v1)
+====
+Adaptive Form Scribble field component written in HTL.
+
+## Features
+
+The Adaptive Form Scribble component allows users to draw or sign directly on a canvas within an adaptive form. It supports the following features:
+* Custom constraint messages for the above types
+* Styles
+
+### Use Object
+The Form Scribble component uses the `com.adobe.cq.forms.core.components.models.form.Scribble` Sling Model for its Use-object.
+
+### Edit Dialog Properties
+The following properties are written to JCR for this Form Scribble component and are expected to be available as `Resource` properties:
+
+1. `./jcr:title` - defines the label to use for this field
```

Comment:
Few edit dialog properties are missing like value fd:dialogLabel

Also, can we add details on fd:viewType and fieldType here, that its view type is signature

Reply:
added